pkg/kp/rc.go

```diff
@@ -0,0 +1,22 @@
+package kp
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+	"github.com/square/p2/pkg/rc"
+)
+
+type ReplicationControllerStore interface {
+	// Create creates a replication controller with the specified manifest and selectors.
+	// The node selector is used to determine what nodes the replication controller may schedule on.
+	// The pod label set is:
+	// 1) applied to every pod the replication controller schedules,
+	// 2) used (in addition to replication controller ID) to determine what pods are already scheduled.
+	// Returns the ID of the replication controller just created, and an error if any.
+	Create(manifest pods.Manifest, nodeSelector labels.Selector, podLabels labels.Set) (string, error)
+	ReplicationController(id string) rc.ReplicationController
```

Comment:
Are these `id string` parameters related to the string that's returned by `Create()`?  Why not have `Create()` return an object on which these methods can be called?